stackhpc_ipa_hardware_managers/system_bios.py

```diff
@@ -114,8 +110,10 @@ def verify_bios_version(self, node, ports):
             LOG.warning('BIOS version verification has been disabled.')
             return True
 
+        vendor_info = hardware.dispatch_to_managers('get_system_vendor_info')
```

Comment:
Will this also get the BIOS version?

Reply:
They don't seem to have a utility meothod to get the bios version. You can get it from the raw `lshw` output though. Mayabe another pull request? To switch over to using lshw?

Reply:
Hmm, I suppose that would be better, or we could just wait and see if there's ever an issue.